examples/common/timestamp/timestamp-with-tz-offset.yaml

```diff
@@ -0,0 +1,13 @@
+---
+id: timestamp-with-tz-offset
+type: Feature
+geometry:
+  type: Point
+  coordinates: [0, 0]
+properties:
+  categories:
+    main: someCategory
+  theme: places
+  type: place
```

Comment:
For variety, maybe make this a different theme/type?

Reply:
Responded up to a similar comment.